data/json/monsters.json

```diff
@@ -4493,7 +4493,7 @@
     "death_function":"NORMAL",
     "special_attacks":[["STARE", 12]],
     "description":"An eyeball the size of an easy chair, covered in rolling blue flames.  It floats through the air, spinning slowly to draw its gaze in every direction.",
-    "flags":["SEES", "WARM", "FLIES", "FIREY", "NO_BREATHE", "NOHEAD", "FAT"]
+    "flags":["SEES", "WARM", "FLIES", "FIREY", "NO_BREATHE", "NOHEAD", "FAT", "HOTAURA"]
```

Comment:
We should *probably* think of some other critter to receive the HOTAURA.  As is, the flaming eye's comparative vulnerability to melee is the only weak point it's got.